recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 # Version should be what `git describe --tags` gives you at the `commit`
-{% set version = "0.3.0.84.a12f7365" %}
+{% set version = "0.3.0.84.ga12f736" %}
```

Comment:
Thanks for fixing this. I'll remove the previous ones which have incorrect version numbers